OData/src/System.Web.Http.OData/OData/Formatter/Serialization/ODataCollectionSerializer.cs

```diff
@@ -10,6 +10,8 @@
 
 namespace System.Web.Http.OData.Formatter.Serialization
```

Comment:
Do you want to change `System.Web.OData`, which is for OData v4?  `System.Web.Http.OData.*` is for OData v3.